src/ecpair.js

```diff
@@ -66,11 +66,12 @@ ECPair.fromWIF = function (string, network) {
 
     network = network.filter(function (network) {
       return version === network.wif
-    }).pop() || {}
+    }).pop()
+    if (!network) throw new Error('Invalid network version')
```

Comment:
I made this because wif version is now optional: https://github.com/bitcoinjs/wif/blob/8355b2bba749de38742ef88f6d8d46992afe7c5e/index.js#L5
We have discussion about this in this issue: https://github.com/bitcoinjs/wif/issues/3